presto-docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -60,6 +62,16 @@ Lambda Functions
                       (s, x) -> CAST(ROW(x + s.sum, s.count + 1) AS ROW(sum DOUBLE, count INTEGER)),
                       s -> IF(s.count = 0, NULL, s.sum / s.count));
 
+.. function:: regexp_replace_lambda(string, pattern, function) -> varchar
```

Comment:
This should go in the regexp section. It’s not a “lambda function”

Reply:
Please see transform and zip_with functions in array.rst. They are functions using lambda. They are all in array.rst but the definition is in lambda.rst.

Reply:
I would say those other functions are misplaced, then. They are array processing functions, not lambda functions.

Reply:
I agree. We should should move the lambda expression part to be part the SQL language chapter, then move the functions to the appropriate pages. (unrelated to this PR)